### DISCUSSION TITLE
Static Filtering in PNP Search Results

### DISCUSSION THREAD

**Discussion by huetvictoria** [microsoft-search/pnp-modern-search#2871] [Q&A] [answered]
Hi, I would like my search results to show only files that meet a certain criteria, where a choice column equals YES. I created a view in the document library so it pre-filters on these files and then added this new view to the Query Template. It is producing 0 results. What am I doing wrong?

{searchTerms} Path:"https://TENANT.sharepoint.com/sites/sitecollection/doclibraryname/Forms/GoldStandard.aspx"

**Comment by kasperbolarsen** [comment, marked as answer]
Hi @huetvictoria 
Your query is looking for content in that specific aspx file, nothing else. I think you should look into a query like {searchTerms} Path:"https://tenant.sharepoint.com/sites/sitecollection/doclibraryname/* ows_[InternalNameOfTheChoiceField]:1

**Reply by huetvictoria** [reply]
Thank you @kasperbolarsen! That didnt quite work but I tailored it a little (see below), so thank you so much for taking the time to respond :)

{searchTerms} Path:https://tenant.sharepoint.com/sites/sitecollection/doclibraryname/ AND RefinableString130:"Yes"

**Reply by kasperbolarsen** [reply]
Happy to hear the issue was resolved.  Feel free to close the discussion.

**Reply by wobba** [reply]
@kasperbolarsen you can now use crawled property names like ows_field in a query. They must me mapped first to a managed property. But if you have site columns, then use the automatically created managed property for filtering. Saves you the mapping.
